subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -158,11 +158,10 @@ endchoice # NRF_CLOUD_PGPS_STORAGE
 if NRF_CLOUD_PGPS_STORAGE_PARTITION
 
 config NRF_CLOUD_PGPS_PARTITION_SIZE
-	int "Size of flash partition for P-GPS data storage"
-	default 86016
+	hex "Size of flash partition for P-GPS data storage"
+	default 0x15000
```

Comment:
I think you might as well just change the default to 0x18000 until the PM bug is fixed, so that applications and samples other than SLM benefit.

If you do that, you can remove the commit which changes `applications/serial_lte_modem/prj.conf`.

Reply:
The workaround is only needed when multiple non-secure storages are needed, so the reason it is needed here is because settings_storage is placed behind it.

Do you still prefer the fix to be applied on the default value?

Reply:
@plskeggs is out of the office, but I think @jayteemo can back me up on this reasoning:

Since it is just a temporary workaround, and the issue affects ATV2 and MMSS as well, and it (hopefully) shouldn't affect anything else negatively, setting it as a default is the Portland team's preferred solution.

Reply:
warning: the value '0x18000' is invalid for NRF_CLOUD_PGPS_PARTITION_SIZE (defined at C:/00_SDK/ncs_main/nrf\subsys\net\lib\nrf_cloud\Kconfig.nrf_cloud_pgps:160), which has type int -- assignment ignored

Should it be decimal 98304 instead?

Reply:
Ok. applied the workaround directly on the Kconfig option itself.